src/pages/paywall/index.tsx

```diff
@@ -1,50 +1,102 @@
-import Card from '@mui/material/Card';
-import CardContent from '@mui/material/CardContent';
-import Typography from '@mui/material/Typography';
-import Stack from '@mui/material/Stack';
 import { UseCaseWrapper } from '../../client/components/common/UseCaseWrapper/UseCaseWrapper';
-import { ARTICLES_SHORTENED, ArticleData } from '../../server/paywall/articles';
+import { ARTICLES, ArticleData } from '../../server/paywall/articles';
 import { CustomPageProps } from '../_app';
 import { USE_CASES } from '../../client/components/common/content';
+import Image from 'next/image';
+import { FunctionComponent } from 'react';
+import styles from './paywall.module.scss';
+import { useRouter } from 'next/router';
+import classNames from 'classnames';
+import { TEST_IDS } from '../../client/e2eTestIDs';
 
-export async function getServerSideProps() {
-  return {
-    props: {
-      articles: ARTICLES_SHORTENED,
-    },
-  };
+/**
+ * Byline helper component
+ */
+
+const BylineDot = () => (
+  <svg xmlns="http://www.w3.org/2000/svg" width="3" height="4" viewBox="0 0 3 4" fill="none">
```

Comment:
Just curios, could we store it as an SVG file and use the `Image` tag?

Reply:
Sure, done! What's the advantage of `<Image>` compared to raw SVG in the DOM btw?

Reply:
In this case, it is not a big. But the main points:
- there are some optimizations in the compiling stage
- `Image` tag will prepare responsive versions for the raster images (not our case)
- `Image` tag can provide lazy loading (not our case again)
- The `Image` tag will force you to add `alt` attribute (generally not an our case again)